pkg/limits/flags.go

```diff
@@ -0,0 +1,138 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license
+
+package limits
+
+import (
+	"flag"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/timescale/promscale/pkg/limits/mem"
+	"github.com/timescale/promscale/pkg/util"
+)
+
+var (
+	MemoryTargetMetric = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: util.PromNamespace,
+			Name:      "max_memory_target_bytes",
+			Help:      "The system will try to keep memory usage below this target",
```

Comment:
This feels less like a description, and more like a conclusion.

Hence how about this?
```suggestion
			Help:      "Maximum memory that can be targeted, in bytes. The system will try to keep memory usage below this target.",
```